File: Pratica 4 - Estrutura de Dados SImples/java_logic/Exercicio1.java
public class Exercicio1 {
    public static void main(String[] args) {
        int [] matriz_A = {2, 4, 6, 8, 10, 12, 14, 16};
        int [] matriz_B = new int[8];
        
        for (int i = 0; i < matriz_A.length; i++) {
            matriz_B[i] = matriz_A[i] * 3;
        }
        System.out.println("Matriz B: ");
        for (int i = 0; i < matriz_B.length; i++) {
            System.out.print(matriz_B[i] + " ");
        }
    }
}

File: Pratica 4 - Estrutura de Dados SImples/cpp_logic/exercicio1.cpp
#include <iostream>
using namespace std;

int main() {
    int matriz_A[8] = {2, 4, 6, 8, 10, 12, 14, 16};
    int matriz_B[8];

    for (int i = 0; i < 8; i++) {
        matriz_B[i] = matriz_A[i] * 3;
    }

    cout << "Matriz B: ";
    for (int i = 0; i < 8; i++) {
        cout << matriz_B[i] << " ";
    }

    return 0;
}